src/handlers/uniswap.ts

```diff
@@ -412,19 +412,20 @@ export const getAllTokens = async () => {
   const { dispatch } = store;
   try {
     let dataEnd = false;
-    let lastId = '';
+    let lastUSDVolume = '1000000000000';
```

Comment:
@skylarbarrera how did we come up with this magic number? can you leave a comment above it explaining what this number is

Reply:
i just set an extremely high number so we're getting all the data, currently the highest USD volume is in the 10's millions so 1 trill should be more than enough. I'll add a comment 😘